src/D_functions_and_procedures.sql

```diff
@@ -362,4 +362,192 @@ $$ LANGUAGE plpgsql;
 
 -- D12 TEST
 SELECT * FROM best_match_titles(ARRAY['apple', 'mads', 'mikkelsen']);
-SELECT title FROM release WHERE media_id=47460;
\ No newline at end of file
+SELECT title FROM release WHERE media_id=47460;
+
+
+
+-------------------D9 SIMILAR MOVIES SEARCH-------------------------
+--------------------------------------------------------------------
```

Comment:
Maybe we need to find a common way to do this function separation so it looks alike in the file.

Reply:
I can make it in the final script